Add JUnit5/Jupiter usage instructions to docs (#954)
testcontainers/testcontainers-java: docs/usage.md

}
```

### Jupiter / JUnit 5

#### Extension

Jupiter integration is provided by means of the `@Testcontainers` annotation.
  
The extension finds all fields that are annotated with `@Container` and calls their container lifecycle 

> methods (only `@Startable` fields are supported). Containers declared as static fields will be shared between test 

Reviewer: > only `@Startable` fields are supported

`Startable` isn't an annotation... Maybe this could be rephrased:

```suggestion
methods (methods on the `Startable` interface). Containers declared as static fields will be shared between test 
```

Author: true, sorry :D
